tomviz/ModuleSlice.cxx

```diff
@@ -48,7 +48,9 @@ namespace tomviz
 {
 
 //-----------------------------------------------------------------------------
-ModuleSlice::ModuleSlice(QObject* parentObject) : Superclass(parentObject)
+ModuleSlice::ModuleSlice(QObject* parentObject)
+  : Superclass(parentObject)
+  , IgnoreSignals(false)
```

Comment:
This looks odd, I think the comma should always be at the end of the line, not the start of the following.